### PR TITLE
Added resourcs for react

### PR DESCRIPTION
**Pull request by karthiknadar1204**
## Changes proposed

Added resources for react js.

### CODE REVIEW

File: database/frontend/react.json [hunk 1]
```diff
@@ -84,5 +84,13 @@
     "url": "https://youtu.be/nDGA3km5He4",
     "category": "frontend",
     "subcategory": "react"
+  },
+  {
+    "name": "React js beginner",
+    "description": "In this hindi video series by Thapa technical, topics ranging from props to react hooks are covered while working on simple projects to gain first hand experience working with react framework.",
```
Comment (CBID2) [inline database/frontend/react.json:90]: [Proofreading] 
```suggestion
    "description": "In this Hindi video series by Thapa technical, topics ranging from props to react hooks are covered while working on simple projects to gain first hand experience with React.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Just added a proofreading suggestion. Other than that, you're off to a great start! :)

**Comment by karthiknadar1204** [comment]
will correct it and push the changes again thank you @CBID2

**Comment by karthiknadar1204** [comment]
@CBID2 Have pushed the changes again, please check now

**Review by karthiknadar1204 — COMMENTED** [review]
Done

**Review by CBID2 — DISMISSED** [review]
Great job @karthiknadar1204! :) You just need one more approved review before merging occurs! :)

**Review by k-deepak04 — DISMISSED** [review]

**Comment by k-deepak04** [comment]
@karthiknadar1204 please resolve the conflicts

**Comment by karthiknadar1204** [comment]
Done @k-deepak04

**Comment by k-deepak04** [comment]
@karthiknadar1204 deployment is failing plz check.

**Comment by karthiknadar1204** [comment]
should i open another pr?

**Comment by CBID2** [comment]
This PR is closed because #1399 solved it.

**Pull request closed without merging**